packages/@ama-sdk/schematics/README.md

```diff
@@ -21,18 +23,17 @@ npx -p @angular/cli ng add @ama-sdk/core
 else
 
 ```shell
-# Install @angular-devkit/schematics-cli, @ama-sdk/core and the SDK generator
-yarn add --dev @angular-devkit/schematics-cli @ama-sdk/schematics @ama-sdk/core
-yarn schematics @ama-sdk/schematics:install
-yarn schematics @ama-sdk/core:install
+npm create @ama-sdk typescript <project-name> -- [--spec-path=./path/to/spec.yaml]
```

Comment:
`--` should be inside the brackets too 
`[-- --spec-path...}`

Reply:
I followed Angular documentation (which is putting it outside)